tests/phpunit/unit/api/term-meta/beans-term-meta/renderFields.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Tests the render_fields method of _Beans_Term_Meta.
+ *
+ * @package Beans\Framework\Tests\Unit\API\Term_Meta
+ *
+ * @since 1.5.0
+ */
+
+namespace Beans\Framework\Tests\Unit\API\Term_Meta;
+
+use Beans\Framework\Tests\Unit\API\Term_Meta\Includes\Beans_Term_Meta_Test_Case;
+use _Beans_Term_Meta;
+use Brain\Monkey;
+
+require_once dirname( __DIR__ ) . '/includes/class-beans-term-meta-test-case.php';
+
+/**
+ * Class Tests_BeansTermMeta_RenderFields
+ *
+ * @package Beans\Framework\Tests\Unit\API\Term_Meta
+ * @group   api
+ * @group   api-term-meta
+ */
+class Tests_BeansTermMeta_RenderFields extends Beans_Term_Meta_Test_Case {
+
+	/**
+	 * Tests _beans_term_meta::render_fields() should output fields HTML when called.
```

Comment:
1. Fix the class name to be `_Beans_Term_Meta::render_fields()`.
2. Remove the `when called` as it's not necessary.

```
Tests _Beans_Term_Meta::render_fields() should render the fields HTML.
```